draft-packages/avatar/KaizenDraft/Avatar/AvatarGroup.module.scss

```diff
@@ -0,0 +1,109 @@
+@import "~@kaizen/design-tokens/sass/typography";
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/color";
+
+$avatar-sm: 1.25rem;
+$avatar-md: 3rem;
+$avatar-lg: 4.5rem;
+
+.AvatarGroup {
+  display: inline-flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.AvatarGroupItem,
+.AvatarCounter {
+  .small & {
+    margin-left: calc(#{$avatar-sm} / -2);
+  }
+
+  .medium & {
+    margin-left: calc(#{$avatar-md} / -2);
+  }
+
+  .large & {
+    margin-left: calc(#{$avatar-lg} / -2);
+  }
+
+  .small & [dir="rtl"] & {
+    margin-left: calc(#{$avatar-sm} / -2);
+  }
+
+  .medium & [dir="rtl"] & {
+    margin-left: calc(#{$avatar-md} / -2);
+  }
+
+  .large & [dir="rtl"] & {
+    margin-left: calc(#{$avatar-lg} / -2);
+  }
+}
+
+.AvatarGroupItem {
+  &:first-of-type {
+    margin-left: 0;
+
+    .small & {
+      [dir="rtl"] & {
+        margin-left: calc(#{$avatar-sm} / -2);
```

Comment:
Again, this looks the same as what's set above?

Reply:
I've been able to remove the previous but not with these. Ideally I'd use something like this (see below) to remove the need the double up but I haven't found a way to use :not with the rtl parent wrapper successfully.

:not([dir="rtl"]) & {
  &:first-of-type {
    margin-left: 0;
  }
}

Reply:
I'm not yet understanding what's going on here, but I wonder if [Logical Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) could help you here at all? i.e. using `margin-inline-start` instead of `margin-left`.

Reply:
I've remove this and added the margin inline styling to the avatar group list. There's still a little bit of duplication but it drops the need to do the deep nesting and I think it's cleaner to read.

Reply:
Nice 😎